### DISCUSSION TITLE
SCAM Website Audacity

### DISCUSSION THREAD

**Discussion by eleminer** [audacity/audacity#884] [Hot potatoes]
Hello guys,
Yesterday I wanted to download audacity.
When entering "audacity" on Google, the following website is the first suggestion: https://www [dot] audacity [dot] de/.
After downloading it from this website, Windows Defender raised the alarm: undesirable behavior.
Here, via Github, I found out that the "correct" homepage has the following URL:
https://www.audacityteam.org/

After downloading it from this website, everything went smoothly and I was able to edit my audio files.

I assume that the first website is a fake?
Is this really the case?

Best regards
Marc

**Comment by SteveDaulton** [comment]
Yes, "audacity.de" is a malware site. We would dearly love to see that site permanently shut down, or at least blocked by Google.

The genuine Audacity website is https://www.audacityteam.org/
Our official downloads are served by https://www.fosshub.com (follow the links from the Audacity website for download /installation instructions and the name of the file that you require). Note that the Audacity download links on FossHub are text links.

**Comment by asa55** [comment]
Reported the malicious site to Google at this link: https://safebrowsing.google.com/safebrowsing/report_badware/?hl=en

More places to report malicious sites are itemized here: https://security.stackexchange.com/questions/1728/where-to-report-malicious-urls-phishing-and-malicious-web-sites

**Comment by eleminer** [comment]
okay, this is very bad....

**Comment by eleminer** [comment]
I'm actually always very careful when downloading files, but this time they got me...

**Comment by stweil** [comment]
The bad news is that the malicious website https://www [dot] audacity [dot] de/ is still listed on the first (!) place when I search for "audacity" at Google. We just detected that we had installed a bundle of new notebooks with malware coming from that site. Today I could confirm that the downloaded Audacity for Windows comes from https://www [dot] audacityorg [dot] de/ and is not identical to the official one, but significantly larger.

I now wrote to Host Europe GmbH (part of the @hosteurope group) who host the scam website and hope they will shut it down and prosecute their client which claims to come from Malta.

I also reported it at @Google using the link above.

As Audacity is a trade mark, it could also be possible to go against domains which misuse that name.

**Comment by SteveDaulton** [comment]
I have modified the direct links to the scam site. We really don't want to be helping their SEO ;-)

**Comment by stweil** [comment]
Symantec just told me that they detect the malicious Windows installer. Windows Defender does not detect it ...

    audacity2-4-2.exe | 7125d68e319e44fe0fe0b6c2412e4ea3 | Already Detected | Trojan.Gen.2 | 209933

**Comment by SteveDaulton** [comment]
I think they change the payload periodically. Much of the time it is a "pup" installer.

**Comment by stweil** [comment]
The owner of the scam website is listed in the [Paradise Papers](https://en.wikipedia.org/wiki/Paradise_Papers): https://offshoreleaks.icij.org/nodes/56039455.

**Comment by DavidBailes** [comment]
I've reported the website to https://www.quad9.net/

**Comment by stweil** [comment]
Host Europe answered today. They ask for evidence that the site is really malicious.

**Comment by SteveDaulton** [comment]
Clicking on the "Download starten" link on this page: https://www [dot] audacity [dot] de/download-de/
downloads this file: https://www.virustotal.com/gui/file/128f53140b13a899bf6304ec2d5db4ebce33df7d3cf5e3386b97a9701a922fed/detection
Tested at 16:54 GMT 16 Feb 2021

**Comment by henricj** [comment]
It might be worthwhile to try to get their code signing certificate revoked.  It will be simple enough for them to get another one, but at least executables that have already been downloaded should get flagged.

https://web.entrust.com/ev-misuse/

I have no idea how responsive Entrust is to these kinds of problems.

**Comment by gwestwood** [comment]
> Symantec just told me that they detect the malicious Windows installer. Windows Defender does not detect it ...

Submit it to https://www.microsoft.com/en-us/wdsi/filesubmission for MS to analyse it.

**Comment by eleminer** [comment]
okay, I was happy a few days ago that the page is down, now it's back online ...

**Comment by stweil** [comment]
Currently they deliver an unmodified audacity-win-3.0.0.exe. But of course one never knows whether they always do that in the future.

**Reply by henricj** [reply]
Just because one person downloads the official release, doesn't mean everybody else will get the same file.

Some of these scam sites are sophisticated enough to provide different downloads depending on what they detect about the downloader.  Geolocation, browser tagging, etc., may lead most folks (likely including those reviewing complaints for a web host), to get a legitimate download, while still letting them target others.

**Comment by eleminer** [comment]
![audacityScam](https://user-images.githubusercontent.com/46229930/114696334-d3069080-9d1c-11eb-91d4-85b2699a0ee6.png)

maybe you should write here that there is a fake page online. As information!

**Reply by jahway603** [reply]
that's a great idea

**Comment by stweil** [comment]
That website has no rights to use the domain name "audacity" without permission. So legal means would be most effective.

**Reply by lengthwave** [reply]
Our legal team is currently working on getting this issue resolved.

**Reply by eleminer** [reply]
perfect. :)

**Reply by eleminer** [reply]
it is still online and via google available. :/

**Reply by differentprogramming** [reply]
The fact that Audacity has a "legal team" is the root of the current problems.

**Reply by Atemu** [reply]
> it is still online and via google available. :/

Things of that nature don't happen nearly as fast as you think they should. Check again in a month or three.

> The fact that Audacity has a "legal team" is the root of the current problems.


No, that's perfectly fine for a software development company. I'd call you crazy if your company didn't have a legal team (or close legal correspondent) above a certain size or user base.

**Comment by eleminer** [comment]
Still online..... what we need to do?

**Comment by lengthwave** [comment]
German law makes this surprisingly time consuming to resolve.

Lawyers are currently working on this and there is an active court case.

**Comment by DePasqualeOrg** [comment]
On Twitter I noticed ads supposedly for Audacity, which point to a website with the .top TLD and otherwise the same domain as the official Audacity website. I just wanted to make a note of this here so that you can follow up on that site as well.

**Reply by eleminer** [reply]
the .top is also SCAM?

**Reply by DePasqualeOrg** [reply]
It looks that way to me. But the Audacity team should follow up on it. The Twitter ads and the profiles that posted them look dubious to me.

**Reply by lengthwave** [reply]
Can you make a screenshot? Do you know the usernames?

**Comment by DePasqualeOrg** [comment]
![twitter-ad](https://github.com/audacity/audacity/assets/25420077/21b0ac6e-197f-418b-adf0-8202f052db2c)

**Reply by DePasqualeOrg** [reply]
This is the ad I've been seeing recently on Twitter, which points to the .top domain with otherwise the same name as the official Audacity domain. I'm assuming it's malware.

**Reply by lengthwave** [reply]
Do you know the domain it is pointing to?

**Reply by DePasqualeOrg** [reply]
It's just as I described it, but I didn't want to write it out here. The official domain is audacityteam.org, right? Just replace .org with .top.

**Reply by eleminer** [reply]
![image](https://github.com/audacity/audacity/assets/46229930/8f6a7c83-5030-45a0-a053-f3195fb1322f)
